cloudformation/parameters.json

```diff
@@ -1,22 +1,32 @@
 [
   {
-    "ParameterValue": "local-development",
-    "UsePreviousValue": false,
-    "ParameterKey": "ResourcePrefix"
+    "ParameterKey": "ProjectName",
+    "ParameterValue": "my-project",
+    "UsePreviousValue": false
   },
   {
-    "ParameterValue": "rate(5 minutes)",
-    "UsePreviousValue": false,
-    "ParameterKey": "ScheduleExpression"
+    "ParameterKey": "EnvironmentName",
+    "ParameterValue": "development",
+    "UsePreviousValue": false
   },
   {
+    "ParameterKey": "ScheduleExpression",
+    "ParameterValue": "rate(24 hours)",
+    "UsePreviousValue": false
+  },
```

Comment:
Two things: 24 hours seems like a weird default value and also, I've no idea why my previous comments preceded the code I was commenting on. Comments should go after, like this one!